.eslintrc

```diff
@@ -11,13 +11,13 @@
 		"func-style": [2, "declaration"],
 		"global-require": 1,
 		"max-lines-per-function": 0,
-		"max-statements-per-line": 1,
+		"max-statements-per-line": [1, {"max": 2}],
```

Comment:
I changed here in order to not change the entire index.js file.
If that is prefered, I can make another PR!